<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpotifyTop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBoxDance">
      <property name="text">
       <string>DANCE</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>SongDetails</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="artist">
         <property name="text">
          <string>artist</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="artistData">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="song">
         <property name="text">
          <string>song</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="songData">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="duration">
         <property name="text">
          <string>duration</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="durationData">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="year">
         <property name="text">
          <string>year</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="yearData">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="popularity">
         <property name="text">
          <string>popularity</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="popularityData">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="genre">
         <property name="text">
          <string>genre</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="genreData">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="loudness">
         <property name="text">
          <string>loudness</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="loudnessData">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="tempo">
         <property name="text">
          <string>tempo</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="tempoData">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="speechiness">
         <property name="text">
          <string>speechiness</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="speechinessData">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="acousticness">
         <property name="text">
          <string>acousticness</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="acousticnessData">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="instrumentalness">
         <property name="text">
          <string>instrumentalness</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="instrumentalnessData">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBoxRock">
      <property name="text">
       <string>ROCK</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>                              SPOTIFY TOP</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxPop">
      <property name="text">
       <string>POP</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" rowspan="2" colspan="2">
     <widget class="QTableView" name="tableView">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBoxHip">
      <property name="text">
       <string>HIP-HOP</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Initial order</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Artist</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Song Name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Genre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Duration</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Popularity</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>SORT BY:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="addNewRowButton">
      <property name="text">
       <string>Add New Row</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Delete Row</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="search">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>search</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="checklabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="actionLOAD"/>
    <addaction name="actionSAVE"/>
   </widget>
   <widget class="QMenu" name="menuINFO">
    <property name="title">
     <string>INFO</string>
    </property>
    <addaction name="actionLOGO"/>
    <addaction name="actionABOUTUS"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuINFO"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSAVE">
   <property name="text">
    <string>SAVE</string>
   </property>
  </action>
  <action name="actionLOAD">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOAD</string>
   </property>
  </action>
  <action name="actionABOUT">
   <property name="text">
    <string>ABOUT</string>
   </property>
  </action>
  <action name="actionLOGO">
   <property name="text">
    <string>LOGO</string>
   </property>
  </action>
  <action name="actionHH">
   <property name="text">
    <string>HH</string>
   </property>
  </action>
  <action name="actionABOUTUS">
   <property name="text">
    <string>ABOUTUS</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
